browser/eb_application.py

```diff
@@ -22,6 +22,11 @@
 app.config.from_object("config")
 
 
+@app.route("/health")
+def healthcheck():
```

Comment:
I think its usually helpful (but not required) to have the route be the same name as the function.